react/week1/todo-list/src/components/Todo.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+
+function Todo(props) {
+    console.log(props);
```

Comment:
Just to develop a good habit you can clear your logs from your final code

Reply:
Thank you for the feedback..
Implemented all the feedbacks